### ISSUE TITLE
Test suite failure with 0.5.4

### ISSUE THREAD

**Issue by snoyberg** [TomMD/DRBG#7]
Available at:

https://gist.github.com/snoyberg/6aff9f7798f4b1358344

**Comment by TomMD** [comment]
I commented on the GIST too:

The CTR DRBG has never passed the NIST tests while the hash and HMAC DRBGs do pass. The reason for this failure is painfully unclear - I might not have understood the terse KAT files correctly or something might not be implemented to specification. I don't have a strong motivation for exploring this right now - if someone is interested then I could consult or I could be bribed to action.

**Comment by snoyberg** [comment]
In that case, should I treat the test suite for DRBG as an expected failure, and not invalidate a Stackage build if it fails?

**Comment by TomMD** [comment]
Yes.  An alternative would be for me or someone else to comment out the CTR tests until they are addressed.
